src/soc/intel/meteorlake/meminit.c

```diff
@@ -38,18 +38,16 @@ static const struct soc_mem_cfg soc_mem_cfg[] = {
 		.num_phys_channels = DDR5_CHANNELS,
 		.phys_to_mrc_map = {
 			[0] = 0,
-			[1] = 1,
```

Comment:
While all changes in this file look reasonable, it breaks existing platforms, doesn't it?

Reply:
I'm not sure. A similar point was raised upstream too: https://review.coreboot.org/c/coreboot/+/82733/1